pages/oss.tsx

```diff
@@ -0,0 +1,299 @@
+    name: "Contracts",
+    description:
+      "Solidity smart contracts that power our ContractKit and Explore.",
+  },
+  {
+    id: "dashboard",
+    name: "Dashboard",
+    description:
+      "User interface to deploy, explore and interact with smart contracts.",
+  },
+  {
+    id: "js",
+    name: "JS Monorepo",
+    description:
+      "TypeScript, React and Wallet SDKs, CLI, Auth, Storage, and more.",
+  },
+  {
+    id: "python-sdk",
+    name: "Python SDK",
+    description: "Create web3 backend applications using Python.",
+  },
+  {
+    id: "unity-sdk",
+    name: "Unity SDK",
+    description: "Create web-based or native web3 games using Unity.",
+  },
+  {
+    id: "go-sdk",
+    name: "Go SDK",
+    description: "Create web3 backend applications using Go.",
+  },
+  {
+    id: "docs",
+    name: "Portal",
+    description: "Documentation, templates, and guides for developers.",
+  },
+  {
+    id: "solana",
+    name: "Solana SDK",
+    description: "Build web3 applications on the Solana blockchain.",
+  },
+  {
+    id: "dynamic-contracts",
+    name: "Dynamic Contracts",
+    description:
+      "Architectural pattern for writing dynamic smart contracts in Solidity.",
+  },
+];
+
+interface RepoCardProps {
+  title: string;
+  description: ReactNode;
+  url: string;
+}
+
+const RepoCard: React.FC<RepoCardProps> = ({ title, description, url }) => {
+  return (
+    <Box
+      as="a"
+      background={"rgba(0,0,0,0.2)"}
+      boxShadow="0 0 0 1px hsl(0deg 0% 100% / 15%)"
+      borderRadius="12px"
+      padding={8}
+      href={url}
+      _hover={{
+        backgroundColor: "rgba(255,255,255,0.03)",
+      }}
+      transition="background-color 0.2s ease"
+    >
+      <Heading as="h3" fontSize="28px" mb={4} fontWeight={700}>
+        {title}
+      </Heading>
+      <Text size="body.lg" lineHeight={1.7} color="#888">
+        {description}
+      </Text>
+    </Box>
+  );
+};
+
+const About: ThirdwebNextPage = () => {
+  const trackEvent = useTrack();
+
+  return (
+    <DarkMode>
+      <Flex
+        sx={{
+          // overwrite the theme colors because the home page is *always* in "dark mode"
+          "--chakra-colors-heading": "#F2F2F7",
+          "--chakra-colors-paragraph": "#AEAEB2",
+          "--chakra-colors-borderColor": "rgba(255,255,255,0.1)",
+        }}
+        justify="center"
+        flexDir="column"
+        as="main"
+        bg="#000"
+      >
+        <HomepageTopNav />
+        <Box mt="-80px" pt="100px" overflowX="hidden">
+          <HomepageSection bottomPattern pb={32}>
+            <Aurora
+              pos={{ left: "50%", top: "0%" }}
+              size={{ width: "2400px", height: "2400px" }}
+              color="hsl(289deg 78% 30% / 45%)"
+            />
+
+            <Flex
+              pt={24}
+              mb={{ base: 24, md: -24 }}
+              flexDir="column"
+              gap={{ base: 6, md: 8 }}
+              align={{ base: "initial", md: "start" }}
```

Comment:
instead of doing this header section from scratch can we use one like we use for product pages? it has most of this already built in

Reply:
ok i changed to use the ProductPage component, not sure if i'm using it correctly - let me know